projects/cdk/package.json

```diff
@@ -18,8 +18,8 @@
         "@ng-web-apis/common": "1.12.1",
         "@ng-web-apis/mutation-observer": "1.1.0",
         "@ng-web-apis/resize-observer": "1.0.3",
-        "@tinkoff/ng-event-plugins": "2.3.2",
-        "@tinkoff/ng-polymorpheus": "3.1.11",
+        "@tinkoff/ng-event-plugins": "2.4.0",
+        "@tinkoff/ng-polymorpheus": "3.1.12",
```

Comment:
please don't forget to update proprietary packages

Reply:
(╯°□°)╯︵ ┻━┻